### PR TITLE
Refactor standard and classification queries

### PR DESCRIPTION
**Pull request by rgwozdz**
This PR is smaller than it looks! There are really only 5 non-test files to review.  It breaks the contents of the old execute-query file into separate files for the `standard` and `classification` query-types.

### CODE REVIEW

File: lib/query/package-features.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  if (groupBy || (!aggregates && !collection)) {
+    return features
+  }
+
+  if (aggregates) {
+    return features[0]
+  }
+
+  if (collection) {
+    return _.chain(collection).cloneDeep().set('features', features).value()
```
Comment (haoliangyu) [inline lib/query/package-features.js:13]: If the `collection` could come with a long array of features, I would suggest dropping the `features` before a deep clone.

File: lib/query/standard-query.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  const { limit } = options
+  const params = createSqlParams(features, options)
+  const filtered = filterAndTransform(sqlString, params)
+
+  if (options.skipLimitHandling || !options.limit) {
+    return packageFeatures(filtered, options)
+  }
+
+  // options.limit is incremented by one in normalizeOptions.js; if filtered.length === options.limit, original limit option has been exceeded
+  const limitExceeded = filtered.length === limit
```
Comment (haoliangyu) [inline lib/query/standard-query.js:16]: I don't understand the logic here. Why is `filtered.length === limit`, not `filtered.length >= limit`?

Comment (rgwozdz) [reply lib/query/standard-query.js:16]: I think that would work fine as well.  But if there was a `limit` option, winnow should have only returned an array of features equal to the `limit` + 1 value.

File: lib/query/package-features.js (hunk 1 above)
Comment (rgwozdz) [reply lib/query/package-features.js:13]: `collection` won't have `features` here.  `features` are removed from any collection wrapper eariler, here https://github.com/koopjs/winnow/pull/184/files/d33032c27e29027f1e85bacd7935776d7417b799#diff-a3fdc082f7c80809c8615ac5bfcc56cb562978d7e571bb4ba6e47e3341ab71bcR24-R29

**Review by haoliangyu — APPROVED** [review]

**Pull request merged**